packages/contracts/contracts/SmartInvoiceFactoryV2.sol

```diff
@@ -0,0 +1,312 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Same comment as for the other copy-paste history break. Versioning via file names is a bad practice.

Reply:
same as above.